internal/handlers/common/aggregations/stages/group.go

```diff
@@ -12,38 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package aggregations
+package stages
```

Comment:
If `countStage` is renamed to `count`, `groupStage` should be just `group`

Reply:
Renamed `groupStage` to `group`